### ISSUE TITLE
Interactive chat with Mathematica notebook - unknown error

### ISSUE THREAD

**Issue by safahim** [WolframResearch/Chatbook#490]
Describe the issue in detail here. Attach any relevant screenshots or files. The section below was automatically generated. Remove any information that you do not wish to include in the report.

<details>
<summary>Debug Data</summary>

| Property | Value |
| --- | --- |
| Name | ``"Wolfram/Chatbook"`` |
| Version | ``"1.0.4"`` |
| EvaluationEnvironment | ``"Session"`` |
| FrontEndVersion | ``"Desktop: 13.3 for Mac OS X x86 (64-bit) (July 24, 2023)"`` |
| KernelVersion | ``"13.3.1 for Mac OS X x86 (64-bit) (July 24, 2023)"`` |
| SystemID | ``"MacOSX-x86-64"`` |
| Notebooks | ``True`` |
| DynamicEvaluation | ``False`` |
| SynchronousEvaluation | ``False`` |
| TaskEvaluation | ``False`` |

## Settings

| Property | Value |
| --- | --- |
| LLMEvaluator | ``"CodeAssistant"`` |

## Stack Data
```
catchTop
throwFailure
trimURL
```

## Failure Data

```
<|
  "Evaluation" :> Wolfram`Chatbook`Actions`Private`writeChunk[Dynamic[Wolfram`Chatbook`Actions`Private`container$66463], CellObject["8914b960-89be-4bd9-8888-413d0bee6416", "7cd5852a-8171-40f6-9d78-d366bf44ae67"], "ta\":{\"role\":\"assistant\",\"content\":\"\"},\"finish_reason\":null}]}\n\n"]
  "Arguments" -> {}
|>
```

</details>

**Comment by raverydavis** [comment]
This may have been resolved in a more recent version please try updating and report back if you still see the same issue. You can update the paclet using `PacletInstall["Wolfram/Chatbook", UpdatePacletSites -> True]`

**Comment by rhennigan** [comment]
This was fixed in #342.

Evaluate the following to get the latest updates:
```
PacletInstall["Wolfram/Chatbook", UpdatePacletSites -> True]
```
